heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -169,6 +169,7 @@ SANDBOX_EXECUTOR_BINARY:             heron.binaries.sandbox.executor
 SANDBOX_STMGR_BINARY:                heron.binaries.sandbox.stmgr
 SANDBOX_TMASTER_BINARY:              heron.binaries.sandbox.tmaster
 SANDBOX_SHELL_BINARY:                heron.binaries.sandbox.shell
+SANDBOX_PYHERON_INSTANCE_BINARY:     heron.binaries.sandbox.pyheron.instance
```

Comment:
probably call 

SANDBOX_PYTHON_HERON_INSTANCE_BINARY and

heron.binaries.sandbox.python.heron.instance